### ISSUE TITLE
Decorative `hr` elements should be hidden from screen reader users

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#21]
# Decorative `hr` elements should be hidden from screen reader users

## Defect level
Feedback will follow a must, should, consider feedback framework:

- ❗️ **Must for if the feedback must be applied.** Failure to follow this feedback may either cause harm to users, block a disabled person from completing an important task, or result in legal risk (fails to meet WCAG 2.1 AA compliance).
- ⚠️ **Should if the feedback is best practice.** Following this feedback will significantly improve the usability for disabled people or reduce the complexity of accessibility barriers.
- ✔️ **Consider for suggestions or enhancements.** Think of these suggestions as reducing the hundreds of papercuts and microaggressive barriers disabled people may face when navigating the web. Going above and beyond here will make us a leader in accessibility. 

## Description
Currently, the decorative `hr` elements will announce out to screen reader users.
<img width="1554" alt="Screen Shot 2022-05-24 at 6 46 10 PM" src="https://user-images.githubusercontent.com/14154792/170144166-519a9e6b-798c-40fe-a323-49141b0b4cd1.png">

Given that these are decorative, and aren't communicating a break in a section, we should consider using a semantic-less element like a `div` instead or add the attribute `aria-hidden`.

## Recommendation or success criteria
- [ ] `div` element is used instead of the `hr` element for decorative elements OR
- [x] decorative `hr` elements have the attribute `aria-hidden="true"`
